Week-01/Day-02_Finding-Nemo/finding_nemo_botiboti/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+//! Week-01/Day-02 solution.
+
+/// Returns the index of the word "Nemo" in the input string or None if "Nemo" is not found.
```

Comment:
```suggestion
/// Returns the index of the word "Nemo" in the words of the input string or None if "Nemo" is not found.
```

Reply:
Though could have been "returns the position of the word "Nemo ..."